plugins/woocommerce/src/Admin/Features/Blueprint/RestApi.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+declare( strict_types = 1);
+
+namespace Automattic\WooCommerce\Admin\Features\Blueprint;
+
+use Automattic\WooCommerce\Blueprint\ExportSchema;
+use Automattic\WooCommerce\Blueprint\ImportSchema;
+use Automattic\WooCommerce\Blueprint\JsonResultFormatter;
+use Automattic\WooCommerce\Blueprint\ZipExportedSchema;
+
+/**
+ * Class RestApi
+ *
+ * This class handles the REST API endpoints for importing and exporting WooCommerce Blueprints.
+ *
+ * @package Automattic\WooCommerce\Admin\Features\Blueprint
+ */
+class RestApi {
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'blueprint';
+
+	/**
+	 * Register routes.
+	 *
+	 * @since 3.5.0
```

Comment:
Not really sure what the correct version should be here. 😅 

```suggestion
	 * @since 9.x.x
```